src/components/ComboBox/ComboBox.scss

```diff
@@ -1,47 +1,50 @@
 $border-color: #eceeef;
 $backrgound-color: #F9FAFB;
 
+$stacking-order: (
+  input: 20,
```

Comment:
@Farhan-emgage typically we don't map for variables that don't exist in the current context. For example, input, backdrop, and spinner are not used in the file. Only icon in used, so that's the only one you need to include. I'll approve this PR so the fix get rolled out this sprint, however please remove the unused map items at some point. Thanks